subsys/config/common/src/app_subGHz_config.c

```diff
@@ -138,6 +135,26 @@ const radio_sx126x_device_config_t radio_sx1262_cfg = {
 
 const radio_sx126x_device_config_t *get_radio_cfg(void)
 {
+	radio_sx1262_cfg.gpio_power =
+		sid_gpio_utils_get_gpio_number_dt(
+			(struct gpio_dt_spec)GPIO_DT_SPEC_GET_OR(DT_NODELABEL(semtech_sx1262_reset_gpios), gpios, INVALID_DT_GPIO));
+	radio_sx1262_cfg.gpio_int1 =
+		sid_gpio_utils_get_gpio_number_dt(
+			(struct gpio_dt_spec)GPIO_DT_SPEC_GET_OR(DT_NODELABEL(semtech_sx1262_dio1_gpios), gpios, INVALID_DT_GPIO));
+	radio_sx1262_cfg.gpio_radio_busy =
+		sid_gpio_utils_get_gpio_number_dt(
+			(struct gpio_dt_spec)GPIO_DT_SPEC_GET_OR(DT_NODELABEL(semtech_sx1262_busy_gpios), gpios, INVALID_DT_GPIO));
+	radio_sx1262_cfg.gpio_rf_sw_ena =
+		sid_gpio_utils_get_gpio_number_dt(
+			(struct gpio_dt_spec)GPIO_DT_SPEC_GET_OR(DT_NODELABEL(semtech_sx1262_antenna_enable_gpios),
+							      gpios, INVALID_DT_GPIO));
+	radio_sx1262_cfg.bus_selector.client_selector =
+		sid_gpio_utils_get_gpio_number_dt(
+			(struct gpio_dt_spec)GPIO_DT_SPEC_GET_OR(DT_NODELABEL(semtech_sx1262_cs), gpios, INVALID_DT_GPIO));
+	radio_sx1262_cfg.bus_selector.speed_hz = DT_PROP_OR(DT_NODELABEL(sid_semtech), clock_frequency, SPI_FREQUENCY_DEFAULT);
+
+	radio_sx1262_cfg.gpio_tx_bypass = GPIO_UNUSED_PIN;
```

Comment:
```suggestion
	radio_sx1262_cfg.gpio_tx_bypass = sid_gpio_utils_get_gpio_number_dt(
			(struct gpio_dt_spec)GPIO_DT_SPEC_GET_OR(DT_NODELABEL(semtech_sx1262_tx_bypass_gpios), gpios, INVALID_DT_GPIO));
```
With this approach user can just define additional `GPIO` in `DTS`, else `INVALID_DT_GPIO` is used. Is it sounds logical?

Reply:
yes that is more elegant solution